src/amo/components/LanguagePicker.scss

```diff
@@ -7,6 +7,8 @@ $selector-padding: 10px;
 }
 
 .LanguagePicker-selector {
+  -moz-appearance: none;
```

Comment:
These weren't being generated locally but I'll double-check to see if I can reproduce that. Maybe it was a weird local config error...